### ISSUE TITLE
im confused by the loss function.

### ISSUE THREAD

**Issue by Hsintao** [eriklindernoren/PyTorch-YOLOv3#96]
in loss function
`loss_cls = (1 / nB) * self.ce_loss(pred_cls[mask], torch.argmax(tcls[mask], 1))`

why calculate the argmax in the axis=1? 
axis =1 means the anchor channel. axis=-1 means the classification channel.

**Comment by Lausannen** [comment]
@Hsintao  when you use tcls[mask], you will get a 0-dimension tensor as a result which can not use axis=1

**Comment by JihaoLee** [comment]
@Lausannen I get a 0-dimension tensor as you say. So there is a problem here. My question is described in detail on the fiftieth issue. What should I do next?  Thank you~

**Comment by eriklindernoren** [comment]
Fixed!
